src/components/SuperHeader.js

```diff
@@ -1,14 +1,29 @@
-import React from "react";
+import React, { Component } from "react";
 import { View } from "react-native";
 import { Time, StyledText as Text } from '.';
-import { withUsername } from '../enhancers';
 import { SuperHeaderStyles as styles } from '../styles';
+import { getItem } from "../util";
 
-const SuperHeader = ({ username }) => (
-  <View style={styles.header}>
-    <Time />
-    <Text style={styles.headerText}>{username}</Text>
-  </View>
-);
 
-export default withUsername(SuperHeader);
+class SuperHeader extends Component {
```

Comment:
Do we want to make this a Pure Component too?

Reply:
Loool. I could